llvm/test/tools/sycl-post-link/device-code-split/indirectly-callable.ll

```diff
@@ -0,0 +1,60 @@
+; RUN: sycl-post-link -split=auto -S < %s -o %t.table
```

Comment:
maybe we could rename these tests to shown the main difference is the split mode? like `_auto_split` and `kernel_split` or something?

Reply:
Good idea, done in 94964b8c5520fc5a046cc1728bdb22c6ac36d2f4